packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -0,0 +1,71 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    IFluidHandle,
+    IFluidHandleContext,
+} from "@fluidframework/core-interfaces";
+import { IDocumentStorageService } from "@fluidframework/driver-definitions";
+import { generateHandleContextPath } from "@fluidframework/runtime-utils";
+
+/**
+ * This class represents blob (long string)
+ * This object is used only when creating (writing) new blob and serialization purposes.
+ * De-serialization process goes through FluidObjectHandle and request flow:
+ * DataObject.request() recognizes requests in the form of `/blobs/<id>`
+ * and loads blob.
+ */
+export class BlobHandle implements IFluidHandle {
+    public get IFluidHandle(): IFluidHandle { return this; }
+
+    public get isAttached(): boolean {
+        return true;
+    }
+
+    public readonly absolutePath: string;
+
+    constructor(
+        public readonly path: string,
+        public readonly routeContext: IFluidHandleContext,
+        public get: () => Promise<any>,
+        public attachGraph: () => void,
+    ) {
+        this.absolutePath = generateHandleContextPath(path, this.routeContext);
+    }
+
+    public bind(handle: IFluidHandle) {
+        throw new Error("Cannot bind to blob handle");
+    }
+}
+
+export class BlobManager {
+    constructor(
+        private readonly routeContext: IFluidHandleContext,
+        private readonly getStorage: () => IDocumentStorageService,
+        private readonly sendBlobAttachOp: (blobId: string) => void,
+    ) { }
+
+    public async getBlob(blobId: string): Promise<BlobHandle> {
+        return new BlobHandle(
+            `blobs/${blobId}`,
```

Comment:
nit: It's a bit cleaner to create new routeContext that would add /blobs/ to "root" routeContext and start using it here (and in createBlob).